src/main/java/controller/RateList.java

```diff
@@ -0,0 +1,98 @@
+package controller;
+
+import model.*;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/9/2019.
+ */
+public class RateList {
```

Comment:
should all methods of RateList class operate with the class member rateList? If not, methods are not bound to the class

Reply:
removed not related methods to new classes - RateListRunner and RateListFilterProperties